ingestion/src/metadata/ingestion/source/dashboard/redash/metadata.py

```diff
@@ -235,11 +235,15 @@ def yield_dashboard_lineage_details(
                     for table in lineage_parser.source_tables:
                         table_name = str(table)
                         database_schema_table = fqn.split_table_name(table_name)
+                        database_schema = database_schema_table.get("database_schema")
+                        database_schema_name = self.check_database_schema_name(
+                            database_schema
+                        )
```

Comment:
We are repeating ourselves in a few places doing similar things. What if we put it into a method to get the table and database schema for a given table name? wdyt?